go/storage/api/metrics.go

```diff
@@ -60,6 +61,19 @@ type metricsWrapper struct {
 	Backend
 }
 
+func (w *metricsWrapper) GetDiff(ctx context.Context, request *GetDiffRequest) (WriteLogIterator, error) {
+	start := time.Now()
+	it, err := w.Backend.GetDiff(ctx, request)
```

Comment:
`GetDiff` function returns an iterator and not a complete result like `SyncGet` function below. Therefore, the metric could return values that are not expected unless we iterate over all write logs (e.g. the next write log is fetched from the database in a background go routine).

Reply:
But this metric is still useful for someone that is aware of this?